packages/babel-plugin/src/css-prop/__tests__/behaviour.test.ts

```diff
@@ -182,15 +182,15 @@ describe('css prop behaviour', () => {
       import '@compiled/react';
 
       const base = { color: 'black' };
-      const top = \` color: red; \`;
+      const top = \` marginTop: 10px; \`;
 
       <div css={[base, top]}>hello world</div>
     `);
 
     expect(actual).toIncludeMultiple([
-      '._syaz5scu{color:red}',
       '._syaz11x8{color:black}',
-      '<div className={ax(["_syaz11x8","_syaz5scu"])}>hello world</div>',
```

Comment:
I never looked into how [composition](https://compiledcssinjs.com/docs/composition#composition) works, but I would have expected only 1 class (`_syaz5scu `) in here in the original example

Reply:
Yea, only 1 class would exist at runtime after `ax` is executed. Previously the babel-plugin wouldn't do any duplication. It just simply constructs something like `ax(["_syaz11x8","_syaz5scu"]` and leave it to runtime `ax` to ensure one atomic group (color) exists.

Reply:
yeahhh, makes sense.

I think I was getting confused by the Styled API behaviour; IIRC it would generate only one class for each property